tests/src/pod-smoke.spec.ts

```diff
@@ -206,5 +207,22 @@ describe.skipIf(process.env.GITHUB_ACTIONS && process.env.RUNNER_OS === 'Linux')
       playExpect(await podsPage.podExists(podToRun)).toBeFalsy();
       await pdRunner.screenshot('pods-pod-deleted.png');
     });
+
+    test('Playing yaml', async () => {
+      const navigationBar = new NavigationBar(page);
```

Comment:
I am not sure if we want to have this test as it is part of pod-smoke. First problem could be the yaml file used, it uses the same images (not a problem) and starts up the same app (name of the pod/container conflict, port conflict ?). 
We definitely want to test this kube. manifest example, just probably in other test class/file. We can keep this test here as well, just please use some different image/app to test - something more lightweight.

Reply:
@odockal I could move the test into a new test file as such it would run independently of any other test, would that be better?

Reply:
Yes that can work! Except I am curious about if it will be working on ubuntu runner on GHA since the pod-smoke does not run properly due to a bug in kernel. Please try it then. We could also check for the image pulled during pod creation as part of that test.

Reply:
moved the test to a new test file